app/Http/Requests/BulkUserCreateRequest.php

```diff
@@ -0,0 +1,72 @@
+<?php
+
+namespace App\Http\Requests;
+
+use Illuminate\Foundation\Http\FormRequest;
+use Illuminate\Validation\Rule;
+
+class BulkUserCreateRequest extends FormRequest
+{
+    /**
+     * Determine if the user is authorized to make this request.
+     *
+     * @return bool
+     */
+    public function authorize()
+    {
+        return true;
```

Comment:
How is this method used?

Reply:
Authorization is already being handled by the middleware that the routes are nested under in routes/api.php, so it doesn't need to be done here.